### DISCUSSION TITLE
Using tailwind via repository inside codesandbox or stackblitz

### DISCUSSION THREAD

**Discussion by Ivluengo** [tailwindlabs/tailwindcss#4049] [Help] [answered]
Hi! Thanks everyone for your help and sorry if this is something not as much advanced as it should, but I'm getting crazy trying to fix this.

I use codesandbox and stackblitz to teach my students how to use tailwind, react, etc. The problem I have is the only way I can use tailwind there is via CDN. I can install the dependencies, but I don't know how to start using it. I tried importing it in the css, I tried creating a postcss and tailwind config files with no result.

Anyone can help me on this? I tried asking in the stackblitz discord channel but no one knows.

 Thanks again for your work!

Ivan

**Comment by dance2die** [comment, marked as answer]
Adam has many sandboxes, https://codesandbox.io/u/adamwathan  
And there are other examples here: https://codesandbox.io/u/adamwathan

Any particular frontend library (such as React, Vue, etc) you are using? Adam's examples use React/Vue there.

**Reply by Ivluengo** [reply]
Hi! Thanks for your answer! Actually I use React, but I want to learn how to use tailwind even in a vanilla JavaScript project. I have seen Adam's examples, but I can't see how he implements tailwind inside his projects apart from using the CDN link inside the public HTML file… The problem with these platforms is we don't have an integrated terminal where to launch scripts, but I don't think it is strictly necessary.  Thanks again!

**Reply by dance2die** [reply]
You can create a sandbox as a "node" project and configure build tools.

I created a template from archived tailwindlab template: https://codesandbox.io/s/tailwind-css-playground-i8o0r
It uses `concurrently` to start both `live-server` and to start Tailwind CSS in watch mode.

Codesanbox provides a way to start the sandbox or the server.

![restart buttons](https://user-images.githubusercontent.com/8465237/114516178-22f53280-9c0b-11eb-9c9b-4b287aa376de.png)

You can run some commands, but not sure the limit on that.

![image](https://user-images.githubusercontent.com/8465237/114516668-9dbe4d80-9c0b-11eb-8d57-c3ff491d439d.png)

**Reply by Ivluengo** [reply]
Hi!! Thanks a lot for your time!! I can see it works in the link you passed. I can see it is kind of hard to make it work...I thought it would be much easier...

Thanks again!
